zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -291,7 +314,10 @@ public Interpreter getCurrentRepl() {
 
     String replName = getRequiredReplName(buffer);
     if (replName != null && cursor > replName.length()) {
-      cursor -= replName.length() + 1;
+      Matcher m = Pattern.compile(String.format("%%%s\\s*", replName)).matcher(buffer);
```

Comment:
I see, should just substring `%replName` and then look in the rest of the string but account for whitespaces?

I'm a bit worry about more issues with parsing `%something` - we have seen quite a bit in the past